doc/man3/OSSL_DECODER_CTX.pod

```diff
@@ -167,12 +167,12 @@ OSSL_DECODER_CTX_get_cleanup() return the values that have been set by
 OSSL_DECODER_CTX_set_construct(), OSSL_DECODER_CTX_set_construct_data() and
 OSSL_DECODER_CTX_set_cleanup() respectively.
 
-OSSL_DECODER_CTX_set_finalized() finalises the context. Functions
-OSSL_DECODER_CTX_set_selection(), OSSL_DECODER_CTX_set_output_type(),
-OSSL_DECODER_CTX_set_output_structure(), OSSL_DECODER_CTX_add_encoder(),
-OSSL_DECODER_CTX_add_extra(), OSSL_DECODER_CTX_set_construct(),
-OSSL_DECODER_CTX_set_construct_data() and OSSL_DECODER_CTX_set_cleanup()
-can't be used after the context is finalised.
+OSSL_DECODER_CTX_set_finalized() finalises the context.
+The functions OSSL_DECODER_CTX_add_decoder(), OSSL_DECODER_CTX_add_extra(),
```

Comment:
This part looks like it belongs in the returns section.

Reply:
Er...  no?  This describes the function, not its return value.

Reply:
(that being said, there's a lot missing in RETURN VALUES)

Reply:
Shane and discussed this briefly, and my impression is that he's in full agreement with @levitte. The function description does need that text, but perhaps more should be said in RETURN VALUES.
Suggestions?  Is that needed in this PR?